doc/whats_new/v0.23.rst

```diff
@@ -106,6 +106,11 @@ Changelog
   :func:`decomposition.non_negative_factorization` now preserves float32 dtype.
   :pr:`16280` by :user:`Jeremie du Boisberranger <jeremiedbb>`.
 
+- |Efficiency| Improved efficiency of :class:`manifold.NMF` when
+  `solver="cd"` by computing the updates of `W` and `H` in parallel.
```

Comment:
Maybe reword to clarify that the parallelism is over samples/features, not over the pair (W, H).

Reply:
I can see the confusion :)
I tried to clarify